presidio-analyzer/pylintrc

```diff
@@ -10,7 +10,7 @@ reports=no
 disable=import-error,too-few-public-methods,missing-docstring,locally-disabled,fixme,cyclic-import,too-many-arguments,invalid-name,duplicate-code
 
 [FORMAT]
-max-line-length=80
+max-line-length=89
```

Comment:
just out of curiosity, why 89?

Reply:
Good question :) just a number...

Reply:
I just got completely sick of the max 80 length